src/Uno.UI/UI/Xaml/Media/Animation/Timeline.animation.cs

```diff
@@ -283,22 +283,32 @@ private void OnAnimatorAnimationEnd(object sender, EventArgs e)
 			private void Play()
 			{
 				_animator?.Dispose();
```

Comment:
@dr1rrb Could this dispose twice now? Maybe I should set `_animator` to null in the new zero duration condition?

Reply:
Don't know if it would have a bad effect, but yes for safety I would reset it to `null`, however I would do it here right after the `Dispose()` (and also in the `OnEnd` to be consistent ^^)

Reply:
This shouldn't be a concern with the last commit since `InitializeAnimator` will always assign a new value to `_animator`